packages/common/src/font.ts

```diff
@@ -232,6 +235,9 @@ export const calculateDynamicFontSize = async ({ textSchema, font, input }: { te
   } else if (dynamicFontSize > dynamicFontSizeSetting.max) {
     dynamicFontSize = dynamicFontSizeSetting.max;
   }
+  if (!('fit' in dynamicFontSizeSetting)) {
+    dynamicFontSizeSetting.fit = DEFAULT_DYNAMIC_FIT;
```

Comment:
in this function, if there is no setting, just use the default value and don't assign it. 

User should not want the data passed by the template to be changed by reference assignment.